client/index.js

```diff
@@ -96,6 +102,13 @@ Matter.Engine.run(engine)
 document.body.addEventListener('keydown', function (e) {
   var key = vkey[e.keyCode]
 
+  //checks if the key begins with < and remove to set the image, because we can't save img with <> on the name.
```

Comment:
In what situations can this happen? `key` should never start with `<` as far as I can tell.